release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -521,6 +521,70 @@ If the following conditions are true, the feature is enabled by default:
+For more information, see xref:../monitoring/config-map-reference-for-the-cluster-monitoring-operator.adoc#metricsserverconfig[MetricsServerConfig] in the config map API reference for the Cluster Monitoring Operator.
+
+[id="ocp-4-15-monitoring-new-feature-to-send-exemplar-data-to-remote-write-storage-for-user-defined-projects"]
+==== New feature to send exemplar data to remote write storage for user-defined projects
+User-defined projects can now use remote write to send exemplar data scraped by Prometheus to remote storage.
+To use this feature, configure remote write using the `sendExemplars` option in the `RemoteWriteSpec` resource.
+For more information, see xref:../monitoring/config-map-reference-for-the-cluster-monitoring-operator.adoc#remotewritespec[RemoteWriteSpec] in the config map API reference for the Cluster Monitoring Operator.
+
+[id="ocp-4-15-monitoring-impoved-alert-querying-for-user-defined-projects"]
+==== Improved alert querying for user-defined projects
+Applications in user-defined projects now have API access to query alerts for application namespaces via the rules tenancy port for Thanos Querier. 
+You can now construct queries that access the `/api/v1/alerts` endpoint via port 9093 for Thanos Querier, provided that the HTTP request contains a `namespace` parameter.
+In previous releases, the rules tenancy port for Thanos Querier did not provide API access to the `/api/v1/alerts` endpoint.
+
+[id="ocp-4-15-monitoring-prometheus-updated-to-tolerate-jitters-at-scrape-time"]
+==== Prometheus updated to tolerate jitters at scrape time
+The default Prometheus configuration in the monitoring stack has been updated so that jitters are tolerated at scrape time.
+For monitoring deployments that have showed sub-optimal chunk compression for data storage, this update helps to optimize data compression, thereby reducing the disk space used by the time series database in these deployments.
+
+[id="ocp-4-15-monitoring-improved-staleness-handling-for-the-kubelet-service-monitor"]
+==== Improved staleness handling for the kubelet service monitor
+Staleness handling for the kubelet service monitor has been improved in order to ensure that alerts and time aggregations are accurate.
+This improved functionality is active by default and makes the dedicated service monitors feature obsolete.
+As a result, the dedicated service monitors feature has been disabled and is now deprecated, and setting the `DedicatedServiceMonitors` resource to `enabled` has no effect.
+
+[id="ocp-4-15-monitoring-improved-ability-to-troubleshoot-reports-of-tasks-failing"]
+==== Improved ability to troubleshoot reports of tasks failing
+The reasons provided when tasks fail in monitoring components are now more granular so that you can more easily pinpoint whether a reported failure originated in components deployed in core platform monitoring, in user workload monitoring, or in both.
```

Comment:
I haven't checked exhaustively but we generally use "user-defined monitoring" instead of "user workload monitoring" in the docs.